### ISSUE TITLE
Annoying debug output from http_archive

### ISSUE THREAD

**Issue by ulfjack** [bazelbuild/bazel#7449]
```
$ bazel build @cockroachdb//:cockroachdb
DEBUG: /private/var/tmp/_bazel_ulfjack/9a270a0c8217952c82652c83ba7012cc/external/bazel_tools/tools/build_defs/repo/http.bzl:43:9: ctx.attr.build_file //dev:cockroachdb.BUILD, path /Users/ulfjack/Projects/example/dev/cockroachdb.BUILD
```

No idea what that is supposed to be good for...

**Comment by ulfjack** [comment]
WORKSPACE is empty except for:
```
http_archive(
    name = "cockroachdb",
    urls = ["https://binaries.cockroachdb.com/cockroach-v2.1.4.darwin-10.9-amd64.tgz"],
    sha256 = "649eb8af1556bba9bd004d54271b8ce364253c4bd511e234345d524b1fa665c8",
    build_file = "//dev:cockroachdb.BUILD",
    strip_prefix = "cockroach-v2.1.4.darwin-10.9-amd64",
)
```

**Comment by aehlig** [comment]
> ```
> $ bazel build @cockroachdb//:cockroachdb
> DEBUG: /private/var/tmp/_bazel_ulfjack/9a270a0c8217952c82652c83ba7012cc/external/bazel_tools/tools/build_defs/repo/http.bzl:43:9: ctx.attr.build_file //dev:cockroachdb.BUILD, path /Users/ulfjack/Projects/example/dev/cockroachdb.BUILD
> ```
> 
> No idea what that is supposed to be good for...

Those were a temporary workaround to avoid redownloads in the restart of the repository rule (#4533) till a proper fix was implemented in 16198bf1f795af34bd72f751cc272c0ecc97f9e0 and cdc99afc1a03ff8fbbbae088d358b7c029e0d232. Those `print` statements have then been removed, the last one in eed98a66405ff70c69d435a9419c3dc4ae7c184a (June 25). Are you sure, you're using a reasonably new version of bazel?

Closing as "fixed upstream". Please reopen with precise version information if the problem persists at head.

**Comment by ulfjack** [comment]
Ugh. My bad, I was actually running a pretty old version of Bazel. Sorry for the noise.
